app/src/app/components/Topbar/MapHeader.tsx

```diff
@@ -1,5 +1,5 @@
 'use client';
-import {Text, Flex} from '@radix-ui/themes';
+import { Flex} from '@radix-ui/themes';
```

Comment:
minor style / spacing fix
can we add this to pre-commit / lint rules?

Reply:
doesn't need to be in this PR, just I have noticed it comes up often

Reply:
Yeah, the precommit should be running prettier but doesn't seem to do so consistently